### PR TITLE
Bump or-tools compat to 9.9

### LINKED ISSUE #2345: Unexpected exception importing solver GLOP and PDLP - new version of ortools
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. 

Hello,
I am doing a python exercise on Jupyter notebook (Anaconda). I need to use the module ContinuousOptimalBinning from optbinning.
After I installed the module of optbinning ('pip install optbinning') using the prompt from Anaconda, I run the python code and I get the following message:

(CVXPY) : Encountered unexpected exception importing solver GLOP:
RuntimeError ('Unrecognized new version of ortools (9.8.29). Expected < 9.8.0 Please open a feature request on cvxpy to enable support for this version.')

 (CVXPY) : Encountered unexpected exception importing solver PDLP:
RuntimeError ('Unrecognized new version of ortools (9.8.29). Expected < 9.8.0 Please open a feature request on cvxpy to enable support for this version.')

Could you please support me on this issue?

Many thanks
Ana

### PR DESCRIPTION
**Pull request by mlubin**
## Description
I tested that Glop and PDLP work without changes with or-tools 9.8 ([release notes](https://github.com/google/or-tools/releases/tag/v9.8))

Closes #2345

## Type of change
- [x] New feature (backwards compatible)
- [ ] New feature (breaking API changes)
- [ ] Bug fix
- [ ] Other (Documentation, CI, ...)

### CODE REVIEW

**Comment by SteveDiamond** [comment]
Looks like GLOP fails on mac for Python 3.11: https://github.com/cvxpy/cvxpy/actions/runs/7381279576/job/20079613727?pr=2316

**Comment by mlubin** [comment]
@mizux we're seeing a segmentation fault in glop on mac with python 3.11, do you have any ideas?

**Comment by mlubin** [comment]
Ping @lperron @mizux for the segfault in glop: https://github.com/cvxpy/cvxpy/actions/runs/7381279576/job/20079613727?pr=2316#step:6:108

**Comment by Mizux** [comment]
hi sorry for the delay, never saw it before....
Is is constantly failing or just flaky ?

Le mer. 3 janv. 2024 à 01:09, Miles Lubin ***@***.***> a
écrit :

**Comment by mlubin** [comment]
> Is is constantly failing or just flaky ?

I'm not sure, I don't personally have a mac to test. 
@SteveDiamond are you able to rerun this CI job?

**Comment by phschiele** [comment]
> > Is is constantly failing or just flaky ?
> 
> I'm not sure, I don't personally have a mac to test. @SteveDiamond are you able to rerun this CI job?

@Mizux @mlubin retriggered the failed test, still failed with the same error.

**Comment by Mizux** [comment]
the MacBookPro I've used to build the python 3.11 wheel is using python 3.11.0 from python.org
your CI seems to use python 3.11.7 (https://github.com/cvxpy/cvxpy/actions/runs/7381279576/job/20296022928?pr=2316#step:6:15)
and from condaforge install `python_abi-3.11` (https://github.com/cvxpy/cvxpy/actions/runs/7381279576/job/20296022928?pr=2316#step:5:77)

Is there an easy way (e.g. one liner) for me to:
1. run the failing test locally to reproduce the error if possible then
2. try to bump my python 3.11 locally and test if it fail
3. regenerate a new wheel against the updated version of python 3.11 to see if it change

note: once I update my local version, I'm not sure I will be able to retrun to previous version -> need to perform all possible tests before upgrading my machine...

**Comment by phschiele** [comment]
Two updates:
- Same config (MacOs, Python 3.11) is still failing after updating this branch
- Replicated the same config locally, and I did not observe the segfault.

**Comment by phschiele** [comment]
@mlubin @SteveDiamond I've bumped from 9.8 to 9.9, and now everything seems to work.

**Review by phschiele — APPROVED** [review]

File: cvxpy/reductions/solvers/conic_solvers/glop_conif.py [hunk 1]
```diff
@@ -51,9 +51,9 @@ def import_solver(self) -> None:
         if Version(ortools.__version__) < Version('9.5.0'):
             raise RuntimeError(f'Version of ortools ({ortools.__version__}) '
                                f'is too old. Expected >= 9.5.0.')
-        if Version(ortools.__version__) >= Version('9.9.0'):
+        if Version(ortools.__version__) >= Version('9.10.0'):
```
Comment (lperron) [inline cvxpy/reductions/solvers/conic_solvers/glop_conif.py:54]: We will release 9.10 this week or next. So maybe you can bump that.

**Pull request merged**